### PR TITLE
[DO NOT MERGE] end point for eliza to get changed uris

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem  @leogrim 

i assume in the returned map, id -> uri pair means oldUriId -> newUri.

subject to requirement changes from Stephen.

### CODE REVIEW

**Comment by yingjieMiao** [comment]
passed test in dev mode

File: kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala [hunk 1]
```diff
@@ -95,6 +95,7 @@ object Shoebox extends Service {
     def logEvent() = ServiceRoute(POST, "/internal/shoebox/logEvent")
     def createDeepLink() = ServiceRoute(POST, "/internal/shoebox/database/createDeepLink")
     def sendPushNotification() = ServiceRoute(POST, "/internal/shoebox/device/sendPushNotification")
+    def getNormalizedUriUpdates(seqNum: Long, limit: Int) =  ServiceRoute(GET, "/internal/shoebox/database/getNormalizedUriUpdates", Param("seqNum", seqNum), Param("limit", limit))
```
Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala]: What exactly does the `limit` here mean? I.e. whats the semantics. How many sequence numbers forward?

Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala]: a.k.a FetchSize

Comment (stkem) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala]: Still don't get it. Is the the maximum number of updates I get, or the maximum number of sequence numbers forwards? Can we just have it be the target sequence number?

Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala]: Sure. I can modify the api.

**Comment by yingjieMiao** [comment]
@stkem 

updated

**Comment by stkem** [comment]
lgtm

**Pull request merged**